pandas/tests/test_generic.py

```diff
@@ -1365,6 +1365,23 @@ def test_spline(self):
         expected = Series([1., 2., 3., 4., 5., 6., 7.])
         assert_series_equal(result, expected)
 
+    def test_spline_extrapolate(self):
+        tm.skip_if_no_package('scipy', '0.15', 'setting ext on scipy.interpolate.UnivariateSpline')
+        s = Series([1, 2, 3, 4, np.nan, 6, np.nan])
```

Comment:
I talked about this here: https://github.com/pydata/pandas/pull/10383#issuecomment-114668678, I can add `kwargs` to `piecewise_polynomial_interpolate` but I am not sure how to test that method.